src/scalarstats.jl

```diff
@@ -570,4 +570,11 @@ Pretty-print the summary statistics provided by `summarystats`:
 the mean, minimum, 25th percentile, median, 75th percentile, and
 maximum.
 """
-describe{T<:Real}(a::AbstractArray{T}) = show(summarystats(a))
+describe(a::AbstractArray) = describe(STDOUT, a)
+describe{T<:Real}(io::IO, a::AbstractArray{T}) = show(io, summarystats(a))
+function describe{T<:Any}(io::IO, a::AbstractArray{T})
```

Comment:
`{T}` is the same as `{T<:Any}`, but you don't actually need the typevar in this case, since you don't care what's in `a`, just that it's an `AbstractArray`, and the typevar isn't referenced in the function body.